com.cinecred.cinecred.yml

```diff
@@ -0,0 +1,41 @@
+id: com.cinecred.cinecred
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: cinecred
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+modules:
+  - name: ghostscript
+    config-opts:
+      - --disable-cups
+      - --disable-dbus
+      - --disable-gtk
+    cleanup:
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10040/ghostscript-10.04.0.tar.gz
+        sha256: c764dfbb7b13fc71a7a05c634e014f9bb1fb83b899fe39efc0b6c3522a9998b1
+  - name: cinecred
+    buildsystem: simple
+    build-commands:
+      - sed -i "s/Icon=cinecred/Icon=$FLATPAK_ID/" cinecred.desktop
+      - sed -i "s/>cinecred/>$FLATPAK_ID/" cinecred.metainfo.xml
+      - mkdir -p /app/share/{applications/,metainfo/} /app/share/icons/hicolor/{scalable/apps/,256x256/apps/}
+      - mv cinecred.desktop "/app/share/applications/$FLATPAK_ID.desktop"
+      - mv cinecred.metainfo.xml "/app/share/metainfo/$FLATPAK_ID.metainfo.xml"
+      - mv cinecred.svg "/app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg"
+      - mv cinecred.png "/app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png"
+      - mv bin/* /app/bin/
+      - mv lib/* /app/lib/
+    sources:
+      - type: archive
+        url: https://cinecred.com/dl/1.7.0/cinecred-1.7.0-linux-x86_64.tar.gz
```

Comment:
is redistribution allowed?

Reply:
I'm the original author and also host the website and downloads, so yes.

Reply:
no aarch64 build? why not building from source?

Reply:
Cinecred is a JVM application with various native components, which have been pre-built and checked into the source tree: https://github.com/LoadingByte/cinecred/tree/master/src/main/natives/linux-x86_64

On Linux, these natives have only been built for x86_64, so compiling the JVM app from source wouldn't suffice to add aarch64 support. At some point in the future, I'll probably check in aarch64 natives, and at that point the Flatpak will also be able to support it.

Reply:
you mean it can't be rebuilt from source? So much for GPL-3.0.

Reply:
They can be rebuilt, and that's documented, but it would make the Flatpak build a lot more complicated: https://github.com/LoadingByte/cinecred/blob/master/MAINTENANCE.md#compiling-native-libraries